packages/net-stubbing/lib/external-types.ts

```diff
@@ -131,6 +131,10 @@ export namespace CyHttpMessages {
      * Request URL.
      */
     url: string
+    /**
+     * URL query string as object.
+     */
+    query: Record<string, string|number>
```

Comment:
hmm, can this truly be a `number`? trying this in Firefox gives me a string:

![image](https://user-images.githubusercontent.com/1151760/122790013-6c1ad000-d2a7-11eb-8cd0-10b151cf2b1e.png)

Reply:
`number` is added because number "input"s should be allowed. Without `number`, the code below is a type error:

```
cy.intercept({ url: '/users*' }, (req) => {
  req.query = {
    a: 42,
  }
})
```

Reply:
I'm not sure of the underlying code, but if `query` is being _deserialized_ from a query string, the values can only ever be strings. If `query` is being used to _serialize_ a query string, the values could really be anything that can be coerced to strings.